pkgs/development/tools/misc/gf/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib, stdenv, makeWrapper
+
+  ,fetchgit, libX11, pkg-config, gdb
+
+  ,freetypeSupport ? false, freetype
+  ,extensionFolder ? null
+}:
+
+let
+  revision = "e0d6d2f59344f853a4a204d5313db6b6a5e5de7d";
+in
+
+stdenv.mkDerivation rec {
+  pname = "gf";
+  version = "2+${builtins.substring 0 7 revision}";
+
+  src = fetchgit {
+    url = "https://github.com/nakst/gf.git";
+    rev = "${revision}";
+    sha256 = "1b9iggz50l9aksblg1lj64daf2l5kjnkixdw8qjjblp63xz668r7";
```

Comment:
I think you were working on a different cached version. This sha doesn't match the checkout.
```
error: hash mismatch in fixed-output derivation '/nix/store/a1sgn19j9w6k37ydf835il48vfiq7r05-gf-e0d6d2f.drv':
         specified: sha256-JyNjfh/m0iUlRrz1OK2chQqnGjGShkeXnipRUP57Ma0=
            got:    sha256-OCo84IgQlkvZrkpYj22ppTecwWXQBsVJxooFazmx1AU=
```

Reply:
Should be fixed now.